### PR TITLE
docs: add section for Android

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR piggybacks off #4536 and makes a few minor changes.

### CODE REVIEW

File: docs/android.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+6. Exit the terminal using `exit` and then reopen the terminal
+7. Install and use Node.js 14:
+
+```shell
+nvm install 14
+nvm use 14
+```
+
+8. Install code-server globally on device with: `npm i -g code-server
```
Comment (code-asher) [inline docs/android.md:21]:
```suggestion
8. Install code-server globally on device with: `npm i -g code-server`
```

**Review by code-asher — CHANGES_REQUESTED** [review]
There is a missing backtick, otherwise :+1:

**Comment by code-asher** [comment]
It occurs to me I should just commit the suggestion, no need to wait for the back and forth :smile:

**Review by code-asher — APPROVED** [review]

**Comment by jsjoeio** [comment]
> It occurs to me I should just commit the suggestion, no need to wait for the back and forth 😄

lol you rock :D

**Pull request merged**